src/components/statusHelper.js

```diff
@@ -343,6 +345,87 @@ export const styledConnectionStatus = (status, err) =>
         </Text>
       </TextContent>
     ),
+    no_rhc: (
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Cannot remediate - Cloud connector not defined
+          <Text component={TextVariants.small} style={{ margin: '0px' }}>
+            Remediation from Insights requires Cloud connector. Cloud connector
+            can be enabled via Satelite, or through
+            <Button
+              style={{ padding: '0px' }}
+              key="configure"
+              variant="link"
+              // eslint-disable-next-line max-len
+              href="LINK HERE"
```

Comment:
@PanSpagetka This link should be a link to this dashboard here:
https://cloud.stage.redhat.com/settings/connector

NOTE:  I grabbed the staging version of this!!!